SUPPORT_POLICY.rst

```diff
@@ -27,11 +27,11 @@ This table describes the current support status of each major version of the AWS
       - 2022-07-08
     * - 2.x
       - General Availability
-      - Maintenance
-      - 2021-07-13
```

Comment:
technically 2.x is in maintenance mode, when we released 3.x i never created the mainline-2.x branch...
we haven't had to do a fix for code on the 2.x version
we should probably create a 2.x branch

Reply:
Oh... fudge... it never ends...
Are all the node modules individually versioned to 3.x?

Reply:
yes, that's not a problem though
we can just check out at this commit https://github.com/aws/aws-encryption-sdk-javascript/tree/05117803a1cc6e0d4fa67845eefabde6d2930980
and push a branch to upstream titled `mainline-2.x`

Reply:
Alright. 
The "breaking change" for 3.x was removal of support for NodeJS 8 and 10.
But, too my knowledge, we are not running tests against 2.x at all...
So... is 2.x in maintenance, or is it in End-of-Support?

Because we have no mechanism validating the package-lock of 2.x is in a good state...

Reply:
you are correct, no tests are run against 2.x, it should be in maintenance

Reply:
Resolved by e45a109